api/config.py

```diff
@@ -87,3 +98,18 @@ async def get_processor_info():
     Returns basic info regarding this processor
     """
     return processor_info(extract_config())
+
+
+@config_router.get("/global_report_emails", response_model=GlobalReportingEmailsInfo)
+async def get_report_emails():
+    return {
+        "emails": extract_config()["App"]["GlobalReportingEmails"]
+    }
+
+
+@config_router.put("/global_report_emails")
+async def get_report_emails(emails: GlobalReportingEmailsInfo, reboot_processor: Optional[bool] = True):
```

Comment:
This function should probably be called 'set...' or 'update...', right?

Reply:
:sweat_smile: